flake.nix

```diff
@@ -0,0 +1,45 @@
+{
+  description = "A list of useful scripts for cardano-node development";
+  inputs = {
+    systems.url = "github:nix-systems/default";
+    haskellNix.url = "github:input-output-hk/haskell.nix";
+    nixpkgs.follows = "haskellNix/nixpkgs-unstable";
+    utils.url = "github:numtide/flake-utils";
+    utils.inputs.systems.follows = "systems";
+  };
+
+  outputs = { nixpkgs, utils, ... }:
+    utils.lib.eachDefaultSystem (system:
+      let
+        pkgs = import nixpkgs { inherit system; };
+        # list of all scripts (without .sh suffix)
+        scripts = with builtins;
+          map (pkgs.lib.removeSuffix ".sh") (attrNames (readDir ./scripts));
```

Comment:
Maybe a good idea to filter out files that are not `.sh`, `removeSuffix` will go ahead even if it doesn't have it

Reply:
Added filtering